addons/html_editor/static/src/core/delete_plugin.js

```diff
@@ -96,7 +96,9 @@ export class DeletePlugin extends Plugin {
         unremovable_node_predicates: [
             (node) => node.classList?.contains("oe_unremovable"),
             // Website stuff?
-            (node) => node.classList?.contains("o_editable"),
+            (node) =>
+                node.classList?.contains("o_editable") &&
+                !this.getResource("banner_container_predicates").some((p) => p(node)),
```

Comment:
I think the whole line about `o_editable` being unremovable in the first place is wrong. You can remove it (including the "Website stuff?" comment) and hopefully it shouldn't break anything.